primitives/api/test/tests/trybuild.rs

```diff
@@ -20,6 +20,11 @@ use std::env;
 #[rustversion::attr(not(stable), ignore)]
```

Comment:
These are probably not needed any longer, as `RUN_UI_TESTS` can be used to toggle the execution.

Reply:
Yeah, but then someone could still run it with a nightly. I would leave it.